lib/sycamore/sycamore/data/table.py

```diff
@@ -275,7 +255,7 @@ def from_html(cls, html_str: Optional[str] = None, html_tag: Optional[Tag] = Non
     # we speculate that duplication may create confusion, so we default to only displaying a cells
     # content for the first row/column for which it is applicable. The exception is for header rows,
     # where we duplicate values to each columnn to ensure that every column has a fully qualified header.
-    def to_pandas(self) -> DataFrame:
+    def to_pandas(self, column_header: Optional[bool] = False) -> Union[DataFrame, List[str]]:
```

Comment:
this type can just be bool with a default. I don't think you need 3 states
also (obnoxious nit) might call the param `column_header_only`